Automattic/wp-calypso
# FHT: Add new checklist to  hosting sites

Closes https://github.com/Automattic/dotcom-forge/issues/4012

## Proposed Changes

* Update ssh checklist item when user creates credential on hosting config
* Add site monitoring checklist item to tasks that are completed when clicked

## Testing Instructions
This PR has to be tested in tandem with https://github.com/Automattic/jetpack/pull/33698 and D126087-code


* See  https://github.com/Automattic/jetpack/pull/33698 for instructions

![image](https://github.com/Automattic/wp-calypso/assets/47489215/8783be77-768f-454f-b7d3-5fd28bdfe9e1)

## Pre-merge Checklist
